{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/analytics/__init__.py

```diff
@@ -0,0 +1 @@
+default_app_config = "{{cookiecutter.repo_name}}.analytics.apps.AnalyticsapiConfig"
```

Comment:
All these files should be if cased around the cookiecutter flag to use analytics